pkg/apis/intevents/v1beta1/doc.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package v1beta1 contains API Schema definitions for the eventing v1beta1 API group
+// +k8s:openapi-gen=true
+// +k8s:deepcopy-gen=package,register
+// +k8s:conversion-gen=github.com/google/knative-gcp/pkg/apis/pubsub
+// +k8s:defaulter-gen=TypeMeta
+// +groupName=pubsub.cloud.google.com
```

Comment:
Should this be updated to `internal.events.*`?

Reply:
Yes it should. Actually switched to base off of `pkg/apis/intevents/v1alpha1/doc.go` (it used to be based on `pkg/apis/pubsub/v1beta1/doc.go`).